browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -298,3 +329,58 @@ IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
   ExpectCookiesOnHost(embedded_test_server(), "a.com", "");
   ExpectCookiesOnHost(embedded_test_server(), "b.com", "");
 }
+
+IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest,
+                       IframeGoogleAccountsCookieAllowed) {
```

Comment:
should these maybe be `ThirdPartyGoogleAccountsCookiesAllowed`? Instead of `Iframe`? I know it's basically equivalent, but I think `ThirdParty` is a better description